### PR TITLE
Starting GCC 2024 blog post :)

### PR DESCRIPTION
**Pull request by EngyNasr**
Hellooo Helloooo <3 

Just wanted to start our GCC 2024 grouped blog post. 

Feel free to take any section, add, edit, or change anything

Thank you,
Engy

### CODE REVIEW

File: content/news/2024-07-01-GCC2024/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+## Thank you to All GCC 2024 sponsors:
+
+* [JXTX Foundation](https://jxtxfoundation.org/)
+* [Worfklow4Metabelomics](https://workflow4metabolomics.org/)
+* [GalaxyWorks](https://galaxyworks.io/)
+* [GigaScience](https://academic.oup.com/gigascience)
+* [KWS](https://www.kws.com/corp/en/)
+* [Limagrain](https://www.limagrain.com/en)
+* [deNBI](https://www.denbi.de/)
```
Comment (bgruening) [inline content/news/2024-07-01-GCC2024/index.md:118]: @erxleben can we here highlight a bit de.NBI and also mention that ESG was present and had a "booth" :)

File: content/news/2024-07-01-GCC2024/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+title: Galaxy Community Conference 2024 - a brief summary!
```
Comment (bgruening) [inline content/news/2024-07-01-GCC2024/index.md:2]: Maybe highlight that this is a Freiburg summary?

Comment (hrhotz) [reply content/news/2024-07-01-GCC2024/index.md:2]: e.g.: "... - a brief summary from the Freiburg Galaxy Team"

File: content/news/2024-07-01-GCC2024/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+![GCC2024 Logo](gcc2024logo.png)
+![GCC2024 Group Picture](gcc2024groupphoto.png)
+
+
+## Keynotes
+
+The conference featured notable keynote speakers, including Jana Klánová from Masaryk University, who discussed the integration of exposomics into biomedical sciences. She highlighted the importance of studying environmental exposures and their effects on health, introduced RECETOX, and discussed European initiatives like PARC and EIRENE that support exposome research. Ute Gunsenheimer, Secretary General of the EOSC Association, spoke on building the EOSC Federation and the potential role of Galaxy. Dr. Daria Onichtchouk from the University of Freiburg shared her extensive use of the Galaxy platform in her research on the genetic control of pluripotency, focusing on the maternal to zygotic transition in zebrafish and its underlying mechanisms, including chromatin accessibility and nucleosome affinity.
+
+## Talks Sessions
```
Comment (hrhotz) [inline content/news/2024-07-01-GCC2024/index.md:29]: I would not sub divide by 'Single Track', 'Analysis Track' and 'Enablement Track'. Just list all the Freiburg talks (no matter which session they were in).

File: content/news/2024-07-01-GCC2024/index.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+During the city tour, we explored the main sights of Brno's city center with guided tours organized by the Brno Tourist Information Centre. We visited the Old Town Hall, St. Peter's Cathedral, as well as the Freedom and Market Squares. The experience was enjoyable and provided a rich insight into Brno's history and culture.
+
+### Conference Nights Activities
+
+Beyond the enriching presentations and discussions, conference nights were filled with fun and camaraderie. Groups of attendees ventured out to the Bouldering Rock Climbing Hall, testing their climbing skills and bonding over shared challenges. Others enjoyed friendly competitions at local billiards halls.
+
+![GCC2024 Billiards](gcc2024billiards.jpg)
+
+Dinner outings were a highlight, with groups exploring Brno’s culinary scene, sampling delicious Mexican and Indian cuisine. The evenings often concluded with open-air talks, where participants relaxed and enjoyed local beers and [Kofola](https://en.wikipedia.org/wiki/Kofola) at various pubs and bars, further strengthening connections and creating memorable experiences.
```
Comment (hrhotz) [inline content/news/2024-07-01-GCC2024/index.md:76]: I would also mention the local cuisine: Schnitzel! @bgruening

Comment (EngyNasr) [reply content/news/2024-07-01-GCC2024/index.md:87]: I couldnot skip the Kofola :D :D <3 <3, I don't know about the Schnitzel story, seems like I missed some of the dinner gatherings :D

File: content/news/2024-07-01-GCC2024/index.md (hunk 3 above)
Comment (EngyNasr) [reply content/news/2024-07-01-GCC2024/index.md:29]: I can do that, once they are written I will just merge them into one

**Comment by bgruening** [comment]
Thanks @EngyNasr!

**Pull request merged**